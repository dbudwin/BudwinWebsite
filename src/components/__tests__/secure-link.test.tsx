import "@testing-library/jest-dom";

import React from "react";
import SecureLink from "../secure-link";
import renderer from "react-test-renderer";

it("creates a link that will open a new tab securely", () => {
    const footer = renderer.create(<SecureLink url={"http://www.budw.in/"} text={"My Website"} />).toJSON();

    expect(footer).toMatchSnapshot();
});
