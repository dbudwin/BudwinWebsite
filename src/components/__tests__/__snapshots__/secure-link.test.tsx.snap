// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`creates a link that will open a new tab securely 1`] = `
<a
  href="http://www.budw.in/"
  rel="noreferrer"
  target="_blank"
>
  My Website
</a>
`;
